global_config.js

```diff
@@ -0,0 +1,35 @@
+// global_config.js
+
+var Parse = require('parse/node').Parse,
+    PromiseRouter = require('./PromiseRouter'),
+    rest = require('./rest');
+
+var router = new PromiseRouter();
+
+// Returns a promise for a {response} object.
+function handleUpdateGlobalConfig(req) {
+  return rest.update(req.config, req.auth,
+                     '_GlobalConfig', 1, req.body)
+  .then((response) => {
+    return {response: response};
+  });
+}
+
+// Returns a promise for a {response} object.
+function handleGetGlobalConfig(req) {
```

Comment:
Right. I've submitted a patch https://github.com/ParsePlatform/parse-server/commit/3b0ab809653dba2b0f47370c563b5cdd8bd0fc5c to my PR